src/layer/place.js

```diff
@@ -339,9 +340,9 @@ export const country1 = {
   type: "symbol",
   paint: {
     "text-color": "#334",
-    "text-halo-blur": 1,
-    "text-halo-color": "rgba(255,255,255,1)",
-    "text-halo-width": 3.0,
+    "text-halo-blur": 0.5,
+    "text-halo-color": Color.backgroundFill,
+    "text-halo-width": 2,
```

Comment:
Did you find that the neutral color was still too loud at a width of 3&nbsp;pixels? It is reasonable to use a wide knockout for emphasis while staying grounded, so to speak. Maybe the blur is actually drawing some attention to these labels.

Reply:
3 pixels just seemed very thick looking when these large country labels stick out over the water.  Over the land 3 pixels does look good though. 

2 pixels:
<img width="419" alt="Screen Shot 2022-12-05 at 10 54 42 PM" src="https://user-images.githubusercontent.com/281482/205809014-90e8eec7-e01f-417b-b8b5-c9b07e41e6f6.png">

3 pixels:
<img width="419" alt="Screen Shot 2022-12-05 at 10 54 08 PM" src="https://user-images.githubusercontent.com/281482/205809047-9564fadd-8cf2-4c3e-88dc-90bf6d396dce.png">

Hmm, yeah maybe the 3 pixels is better than 2 even when it appears to create a peninsulas 😀

Reply:
How about without a blur? I know it sounds counterintuitive, but maybe the blur is drawing extra attention to the labels that wouldn’t be as pronounced if the halo were simply wide.

Reply:
No blur looks pretty good.  Definitely a much sharper look (on a 2x display anyway), though.  

<img width="415" alt="Screen Shot 2022-12-05 at 11 39 29 PM" src="https://user-images.githubusercontent.com/281482/205816923-d366dbef-fa56-402d-91e6-5bb5da4bbfd8.png">

To make the halos as thick as possible, no blur may be necessary.  It seems that once the halo or it's blur hits the edge of the letter box, we start getting blocky artifacts.
https://osmus.slack.com/archives/C01V02K52UX/p1670300775628049?thread_ts=1670296592.027569&cid=C01V02K52UX

Reply:
I like this look without the blur. But if you think it’s too eye-catching, we could make the halo color slightly translucent, with an alpha of, say, 90%. It wouldn’t quite be a knockout anymore, but any water or roads that shine through would be less apparent.